test/functional/test-impression.js

```diff
@@ -199,7 +200,7 @@ describe('impression', () => {
     viewer.getParam.withArgs('click').returns('https://www.example.com');
     sandbox.stub(viewer, 'sendMessageAwaitResponse').callsFake(message => {
       if (message == 'getReplaceUrl') {
-        return Promise.resolve();
+        return Promise.resolve({});
```

Comment:
We will either reject the promise or return an object with a { 'replaceUrl': ___ } field. This field may be undefined or it may be a string containing the absolute replace URL.

Reply:
Changed to `{'replaceUrl': undefined}` here.